scripts/install-git-hooks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+
+MODE="${1:-"all"}"
+
+mkdir -p "$(git rev-parse --git-dir)/hooks"
+
+if [[ "$MODE" == "all" || "$MODE" == "signoff" ]]; then
+  cp scripts/signoff-git-hook.sh "$(git rev-parse --git-dir)/hooks/commit-msg"
```

Comment:
Thanks for choosing `git rev-parse --git-dir` instead of `.git`!

![](https://media.giphy.com/media/XbxZ41fWLeRECPsGIJ/giphy.gif)

Reply:
That's the only thing that correctly works with workspaces :smile: